### PR TITLE
Translation: Polish

### PR DESCRIPTION
**Pull request by mbiesiad**
Hi. First of all, great & interesting project! I saw that Polish version is uncomplete, not finished yet. So, here (eg. see the diff comparison):
https://github.com/mbiesiad/coding-interview-university/blob/master/translations/README-pl.md

@jwasham  
is updated & finished Polish version. 🔥  Please, enjoy!
Best greetings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 - [Tiếng Việt - Vietnamese](translations/README-vi.md)
 - [Español](translations/README-es.md)
 - [Português Brasileiro](translations/README-ptbr.md)
+- [Polish](https://github.com/mbiesiad/coding-interview-university/blob/master/translations/README-pl.md)
```
Comment (jwasham) [inline README.md:22]: You can remove the full URL and just do translations/README-pl.md

Comment (mbiesiad) [reply README.md]: Sure, done. Thank you!

**Comment by mbiesiad** [comment]
> 
> 
> Wow, this is great!

My pleasure! 🏅

**Comment by jwasham** [comment]
Thanks for the hard work, Michal!

**Pull request merged**